public/app/features/admin/ldap/LdapSettingsPage.tsx

```diff
@@ -266,7 +271,7 @@ export const LdapSettingsPage = () => {
                   <Trans i18nKey="ldap-settings-page.title">Basic Settings</Trans>
                 </h3>
                 <Field
-                  label={t('ldap-settings-page.host.label', 'Server host')}
+                  label={t('ldap-settings-page.host.label', 'Server host *')}
```

Comment:
Please keep the same format introduced here with `Server host *` for the `Search filter*` field as well. In other words, the label for Search filter should be `Search filter *` instead of `Search filter*` (add a space before * char).

Reply:
Changed as requested